vscode-extension/src/extension.ts

```diff
@@ -38,9 +41,12 @@ export function activate(context: vscode.ExtensionContext) {
     log: true,
   });
 
-  const setupCommand = vscode.commands.registerCommand(COMMANDS.INIT, () => {
-    installDependencies(context, extensionOutputChannel);
-  });
+  const setupCommand = vscode.commands.registerCommand(
+    COMMANDS.INIT,
+    async () => {
+      await initialize(context, extensionOutputChannel);
```

Comment:
Does this actually need do async/await? I think it can just fire off the call to initialize since nothing else is waiting for it

Reply:
not needed, added that while trying things out, removed